runtime/src/invoke_dpe.rs

```diff
@@ -23,21 +27,44 @@ impl InvokeDpeCmd {
             let hashed_rt_pub_key = crypto
                 .hash(AlgLen::Bit384, &rt_pub_key.to_der()[1..])
                 .map_err(|_| CaliptraError::RUNTIME_INITIALIZE_DPE_FAILED)?;
+            let image_header = pdata.manifest1.header;
```

Comment:
ImageHeader implements copy, so there's a chance this would be copied unnecessary. We should either

* Reference members as needed
* Make this local variable a borrow (`let image_header = &pdata.manifest.header`)

Reply:
Done.